dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Dtos/UserDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Devon4Net.Application.WebAPI.Implementation.Domain.Entities;
+using System.ComponentModel.DataAnnotations;
+
+namespace Devon4Net.Application.WebAPI.Implementation.Business.SessionManagement.Dto
```

Comment:
To be consistent with the rest of the code, the namespace should be ...Dto**s** (plural)

Reply:
Thanks, added the s!